tests/test_crypto.py

```diff
@@ -1,5 +1,8 @@
 import unittest
-from unittest import mock
+try:
```

Comment:
This change is unnecessary as long as backports.unittest_mock is installed, which it should be by tox and tests/requirements.